crates/sui-indexer-alt/migrations/2024-10-21-003426_tx_indices/up.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS tx_affected_addresses
+(
+    affected                    BYTEA        NOT NULL,
+    tx_sequence_number          BIGINT       NOT NULL,
+    sender                      BYTEA        NOT NULL,
+    PRIMARY KEY(affected, tx_sequence_number)
+);
+
+CREATE INDEX IF NOT EXISTS tx_affected_addresses_tx_sequence_number ON tx_affected_addresses (tx_sequence_number);
+CREATE INDEX IF NOT EXISTS tx_affected_addresses_sender ON tx_affected_addresses (sender, affected, tx_sequence_number);
+
+CREATE TABLE IF NOT EXISTS tx_digests
+(
+    tx_digest                   BYTEA        PRIMARY KEY,
+    tx_sequence_number          BIGINT       NOT NULL
+);
+CREATE INDEX IF NOT EXISTS tx_digests_tx_sequence_number ON tx_digests (tx_sequence_number);
```

Comment:
This should still be like this, I think:

```suggestion
CREATE TABLE IF NOT EXISTS tx_digests
(
    tx_sequence_number          BIGINT       PRIMARY KEY,
    tx_digest                   BYTEA        NOT NULL
);
```